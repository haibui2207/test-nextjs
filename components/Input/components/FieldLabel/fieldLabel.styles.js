import { createUseStyles } from 'react-jss';

import COLORS from '../../../../constants/COLORS';
import { hexToRgbA, rem } from '../../../../utils/jss';

export default createUseStyles({
  root: {
    display: 'inline-block',
    fontWeight: 500,
    fontSize: rem(12),
    lineHeight: rem(16),
    color: hexToRgbA(COLORS.textBlack, 0.8),
    marginBottom: rem(8),
    fontFamily: 'inherit',
    '&.required:after': { fontSize: rem(16), content: '" *"', color: 'red' },
  },
});
