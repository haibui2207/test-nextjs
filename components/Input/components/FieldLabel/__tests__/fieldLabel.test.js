import React from 'react';
import { shallow } from 'enzyme';

import FieldLabel from '../FieldLabel';

describe('Testing Input component > FieldLabel', () => {
  it('should render match snapshot', () => {
    const wrapper = shallow(<FieldLabel htmlFor="something" label="Label" />);
    expect(wrapper).toMatchSnapshot();
  });
});
