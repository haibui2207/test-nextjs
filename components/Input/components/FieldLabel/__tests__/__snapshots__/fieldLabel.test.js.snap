// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Input component > FieldLabel should render match snapshot 1`] = `ShallowWrapper {}`;
